drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/editor/RuleModeller.java

```diff
@@ -481,16 +479,20 @@ protected void showConditionSelector(Integer position) {
         popup.show();
     }
 
-    protected void showActionSelector(Widget w,
-                                      Integer position) {
-        RuleModellerActionSelectorPopup popup = new RuleModellerActionSelectorPopup(model,
-                                                                                    this,
-                                                                                    actionPlugins,
-                                                                                    position,
-                                                                                    getDataModelOracle());
+    protected void showActionSelector(final Integer position) {
+        final RuleModellerActionSelectorPopup popup = ruleModellerActionSelectorPopup(position, actionPlugins);
         popup.show();
     }
 
+    protected RuleModellerActionSelectorPopup ruleModellerActionSelectorPopup(final Integer position,
+                                                                              final Collection<RuleModellerActionPlugin> testableActionPlugins) {
```

Comment:
Do you really want to call this ```testableActionPlugins``` since this is not test code?

Reply:
Updated to standard name.